babel.config.js

```diff
@@ -31,24 +31,32 @@ const argv = minimist(process.argv.slice(2));
 
 // eslint-disable-next-line amphtml-internal/no-module-exports
 module.exports = function(api) {
+  const isDist = argv._.includes('dist');
+  const {esm} = argv;
+  const es5Target = {
+    'browsers': isTravisBuild()
+      ? ['Last 2 versions', 'safari >= 9']
+      : ['Last 2 versions'],
+  };
+
+  const esmTarget = {
```

Comment:
Nit: Is constant, and can hoist this if you want out of the current function.

Reply:
done